customer_team_manager/__manifest__.py

```diff
@@ -36,7 +36,7 @@
             "/customer_team_manager/static/src/xml/field_widget_template.xml",
             "/customer_team_manager/static/src/css/export.css",
             "/customer_team_manager/static/src/scss/field_widget.scss",
-            "/customer_team_manager/static/src/js/field_widget.js",
+            "/customer_team_manager/static/src/js/field_widget.esm.js",
```

Comment:
Le message de commit est un peu trop verbeux et contient au moins une typo (to better => to improve).

Je suis pour des messages de commit bien explicites mais peut-être un peu plus efficaces.